scripts/check-licenses.js

```diff
@@ -0,0 +1,114 @@
+import * as fs from "node:fs";
+import * as path from "node:path";
+import * as process from "node:process";
+import * as url from "node:url";
+
+// ------------
+// License list
+// ------------
+
+const allowedLicenses = [
+	"BSD",
+	"0BSD",
+	"Apache-2.0", "Apache 2.0",
+	"Artistic-2.0",
+	// TODO: uncomment, this is only commented for demonstration purposes
+	//"BSD-2-Clause",
+	//"BSD-3-Clause",
```

Comment:
This should be reverted prior to merging:

```suggestion
	"BSD-2-Clause",
	"BSD-3-Clause",
```